### ISSUE TITLE
Panic when vault identity group has no member entities

### ISSUE THREAD

**Issue by anth0d** [hashicorp/terraform-provider-vault#1136]
### Terraform Version
```0.13.4```

### Affected Resource(s)
Please list the resources as a list, for example:
- `vault_identity_group_member_entity_ids`

### Terraform Configuration Files
```hcl
data "vault_identity_group" "this" {
  group_name = var.group
}

resource "vault_identity_entity" "this" {
  name     = "${var.group}-entity"
}

resource "vault_identity_group_member_entity_ids" "this" {
  group_id          = data.vault_identity_group.this.id
  member_entity_ids = [vault_identity_entity.this.id]
  exclusive         = false
}
```

### Debug Output
https://gist.github.com/anth0d/3dc7d1a9ce13083d9420762abfd48c74

### Panic Output
Not available

### Expected Behavior
Handle empty entity list

### Actual Behavior
Panic during interface conversion

### Steps to Reproduce
1. create a group in vault
2. delete any entities from the group
3. run `terraform apply`

### Important Factoids
None

### References
None

**Comment by tvoran** [comment]
Hi @anth0d, I haven't been able to reproduce this since I always see `member_entity_ids` present in the response, even when it's an empty list. I'm curious what version of vault you're using?

**Comment by angeloskaltsikis** [comment]
We have the same issue with Vault `1.8.0`

**Comment by anth0d** [comment]
@tvoran actually we are using Vault `1.6.2`. Sorry I did not see any notification for this mention!

**Comment by candonov** [comment]
Same issue with Vault `1.9.0`

**Comment by vinay-gopalan** [comment]
Thanks for opening this issue! The nil handling case for member entities was included in #1409 and was released as part of TFVP v3.5.0. Closing this issue, but if you are experience any other problems with this resource then please do let us know, thanks once again!
